packages/utilities-react/src/StepFlow/types.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Dispatch, SetStateAction } from 'react';
+
+export type StepContextType = StepState;
+
+// This interface should be extended by the consumer to match
+// their own unique use case given the fields within their
+// own stepped experience
```

Comment:
Suggestion to use a JSDoc comment since the code is documenting a type.

```suggestion
/**
 * This interface should be extended by the consumer to match their own unique
 * use case given the fields within their own stepped experience.
 */
```